pkg/cli/bicep/parameters_test.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2023 The Radius Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bicep
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_ExtractParameters(t *testing.T) {
+	b, err := os.ReadFile("testdata/test-extractparameters.json")
+	require.NoError(t, err)
+
+	template := map[string]any{}
+	err = json.Unmarshal(b, &template)
+	require.NoError(t, err)
+
+	params, err := ExtractParameters(template)
+	require.NoError(t, err)
+
+	expected := map[string]any{
+		"application": map[string]any{
```

Comment:
Should we try to see if it fails with a capital letter (Application)?

Reply:
This code path doesn't compare parameter names, it just reads them.